app/src/organisms/EmergencyStop/EstopPressedModal.tsx

```diff
@@ -23,6 +23,8 @@ import {
 } from '@opentrons/components'
 
 import { useAcknowledgeEstopDisengageMutation } from '@opentrons/react-api-client'
+import { usePlacePlateReaderLid } from '/app/resources/modules'
+
 
```

Comment:
```suggestion

import { usePlacePlateReaderLid } from '/app/resources/modules'
```
Nit, we generally place local imports in a grouping separate from the monorepo imports.